packages/juce-blueprint/src/lib/NativeMethods.ts

```diff
@@ -1,8 +1,9 @@
 
+//@ts-ignore
```

Comment:
Yea... this file should maybe go away too. It was meant to protect methods installed from native, but the user will obviously be able to figure out for themselves if their native method installation is broken